airbyte-api/src/main/openapi/config.yaml

```diff
@@ -3296,9 +3302,24 @@ components:
           description: The query parameters present in the redirect URL after a user granted consent e.g auth code
           type: object
           additionalProperties: true # Oauth parameters like code, state, etc.. will be different per API so we don't specify them in advance
-    CompleteOauthResponse:
+        connectionConfiguration:
```

Comment:
How come we are passing in `connectionConfiguration` and not an oauth specific configuration like we discussed in the spec?

Reply:
if we pass the `connectionConfiguration` (which includes the `authConfiguraton` as a child node)

we can do the "seeding" to the connectionConfiguration in the backend and return the modified config back to the frontend.

We can also go read the "input" fields such as `shop_id` directly in the connectionConfiguration

Reply:
The second option where the "seeding" is done by the UI would be from this other alternative PR: https://github.com/airbytehq/airbyte/pull/7798 (then we can close and not merge this one)